src/internal/observable/race.ts

```diff
@@ -31,7 +31,18 @@ export function race<T>(...observables: ObservableInput<T>[]): Observable<T>;
 export function race(...observables: ObservableInput<any>[]): Observable<{}>;
 
 /**
- * Returns an Observable that mirrors the first source Observable to emit an item.
+ * Returns an observable that mirrors the first source observable to emit an item.
+ *
+ * ![](race.png)
+ *
+ * `race` returns an observable, that when subscribed to, subscribes to all source observables immediately.
+ * As soon as one of the source observables emits a value, the result unsubscribes from the other sources.
+ * The resulting observable will forward all emissions, including error and completion, from the "winning"
+ * source observable.
+ *
```

Comment:
One thing that I would add it the situation of errors of source observables. 
If one of the used sources errors before a first notification the race operator also errors no matter if another source could potentially win the race.

Reply:
Thanks for suggession. I have made the changes as suggested.